packages/flutter_tools/lib/src/commands/packages.dart

```diff
@@ -399,14 +399,7 @@ class PackagesGetCommand extends FlutterCommand {
     return findPlugins(rootProject, throwOnError: false);
   })();
 
-  late final String? _androidEmbeddingVersion = (() {
-    final FlutterProject? rootProject = _rootProject;
-    if (rootProject == null) {
-      return null;
-    }
-
-    return rootProject.android.getEmbeddingVersion().toString().split('.').last;
-  })();
+  late final String? _androidEmbeddingVersion = _rootProject?.android.getEmbeddingVersion().toString().split('.').last;
```

Comment:
I really wonder why this was done in a self calling function in the first place... Maybe just to limit the scope of the variables?